src/analysis/normed_space/basic.lean

```diff
@@ -905,10 +910,16 @@ end
 @[simp] lemma nnnorm_eq_zero {a : α} : ∥a∥₊ = 0 ↔ a = 0 :=
 by simp only [nnreal.eq_iff.symm, nnreal.coe_zero, coe_nnnorm, norm_eq_zero]
 
+/-- An injective group homomorphism from an `add_comm_group` to a `normed_group` induces a
+`normed_group` structure on the domain. -/
+def normed_group.induced [add_comm_group γ]
+  (f : γ →+ α) (h : function.injective f) : normed_group γ :=
+{ .. semi_normed_group.induced f,
+  .. metric_space.induced f h normed_group.to_metric_space, }
```

Comment:
It's unfortunate that the norm-less version of this is called `function.injective.add_comm_group`. Would it be sensible to expose this as an an alias with a similar name?

Reply:
I'll follow up with an alias and merge this for now.